### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by VincentLanglet**
Until we move this doc to a doc folder, this is still better than what we have currently.

### CODE REVIEW

**Review by jordisala1991 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -29,39 +54,35 @@ points in time.
...
-## Installation (In Symfony2 Application)
+## Installation
 
 ### Installing the bundle
 
 Simply run assuming you have installed composer.phar or composer binary:
 
 ``` bash
-$ php composer.phar require simplethings/entity-audit-bundle
+$ php composer.phar require sonata-project/entity-audit-bundle
```
Comment (phansys) [inline README.md:64]:
```suggestion
$ php composer.phar require sonata-project/entity-audit-bundle
```
```suggestion
$ composer require sonata-project/entity-audit-bundle
```

File: README.md [hunk 2]
```diff
@@ -29,39 +54,35 @@ points in time.
...
 ```
 
 ### Configuration
 
-Load extension "simple_things_entity_audit" and specify the audited entities (yes, that ugly for now!)
+Load extension "simple_things_entity_audit" and specify the audited entities
 
-##### app/config/config.yml
+##### config/entity_audit.yml
 ```yml
```
Comment (phansys) [inline README.md:86]:
```suggestion
```yml
# config/packages/entity_audit.yaml

```

File: README.md [hunk 3]
```diff
@@ -70,7 +91,7 @@ simple_things_entity_audit:
 ```
 If you need to exclude some entity properties from triggering a revision use:
 
-##### app/config/config.yml
+##### config/entity_audit.yml
 ```yml
```
Comment (phansys) [inline README.md:95]:
```suggestion
```yml
# config/packages/entity_audit.yaml

```

File: README.md [hunk 4]
```diff
@@ -218,18 +231,19 @@ $revision = $auditReader->getCurrentRevision(
...
-In the Symfony2 web context the username is resolved from the one in the current security context token.
+In the Symfony web context the username is resolved from the one in the current security context token.
 
-You can override this with your own behaviour by configuring the `username_callable` service in the bundle configuration. Your custom service must be a `callable` and should return a `string` or `null`.
+You can override this with your own behaviour by configuring the `username_callable` service in the bundle configuration.
+Your custom service must be a `callable` and should return a `string` or `null`.
 
-##### app/config/config.yml
+##### config/entity_audit.yml
 ```yml
```
Comment (phansys) [inline README.md:240]:
```suggestion
```yml
# config/packages/entity_audit.yaml

```

File: README.md [hunk 5]
```diff
@@ -241,12 +255,12 @@ $auditConfig->setUsernameCallable(function () {
...
 
-A default Symfony2 controller is provided that gives basic viewing capabilities of audited data.
+A default Symfony controller is provided that gives basic viewing capabilities of audited data.
 
 To use the controller, import the routing **(don't forget to secure the prefix you set so that
 only appropriate users can get access)**
 
-##### app/config/routing.yml
+##### config/entity_audit.yml
 ```yml
```
Comment (phansys) [inline README.md:264]:
```suggestion
```yml
# config/packages/entity_audit.yaml

```

**Review by phansys — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -29,39 +54,35 @@ points in time.
...
+];
 ```
 
 ### Configuration
 
-Load extension "simple_things_entity_audit" and specify the audited entities (yes, that ugly for now!)
+Load extension "simple_things_entity_audit" and specify the audited entities
 
-##### app/config/config.yml
+##### config/packages/entity_audit.yml
```
Comment (phansys) [inline README.md:85]: IMO, we must place these paths as a comment inside the code snippet, like in the previous snippet for `config/bundles.php`. This way, we will be consistent with the rest of [our packages](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/docs/reference/configuration.rst#configuration-1).

Comment (VincentLanglet) [reply README.md]: Done

File: README.md [hunk 7]
```diff
@@ -29,57 +54,55 @@ points in time.
...
     audited_entities:
         - MyBundle\Entity\MyEntity
         - MyBundle\Entity\MyEntity2
 ```
 If you need to exclude some entity properties from triggering a revision use:
 
-##### app/config/config.yml
-```yml
+```yaml
+# config/packages/entity_audit.yml
```
Comment (phansys) [inline README.md:96]: We usually leave a LF between the commented path and the file contents.
```suggestion
# config/packages/entity_audit.yml

```

Comment (phansys) [inline README.md:96]: IMO, we could directly enforce the `.yaml` extension.
```suggestion
# config/packages/entity_audit.yaml
```

Comment (VincentLanglet) [reply README.md]: Done

Comment (VincentLanglet) [reply README.md]: Indeed, done

**Review by phansys — APPROVED** [review]

**Pull request merged**